browser.c

```diff
@@ -1011,7 +1021,10 @@ tabbed_launch(void)
     GIOChannel *tabbed_stdout_channel;
     GError *err = NULL;
     gchar *output = NULL;
-    char *argv[] = { "tabbed", "-c", "-d", "-p", "s1", "-n", __NAME__, NULL };
+    char geom[8];
```

Comment:
`g_strdup_printf` does the job: https://developer.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-strdup-printf :)